.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,26 @@
+name: Unit Tests
+on:
+  workflow_dispatch: null
+  push:
+  pull_request:
+jobs:
+  unit_tests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          ref: dev
+
+      - name: Checkout code
+        uses: actions/setup-go@v4
+        with:
+          go-version: '1.19'
+
+      - run: go version
+
+      - name: Run smoke tests
+        run: make unittest
+        env:
+          LINODE_TOKEN: ${{ secrets.DX_LINODE_TOKEN }}
```

Comment:
Could this secret be dropped so it's not accidentally exposed in PR workflow runs?

Reply:
Yup I will remove this